core/templates/dev/head/components/state-editor/state-responses-editor/state-responses.directive.ts

```diff
@@ -236,6 +252,17 @@ oppia.directive('stateResponses', [
             return interactionId && INTERACTION_SPECS[interactionId].is_linear;
           };
 
+          $scope.isCurrentInteractionTrivial = function() {
+            var interactionId = $scope.getCurrentInteractionId();
+            if (
```

Comment:
Can simplify all this to `return INTERACTION_IDS_WITHOUT_ANSWER_DETAILS.indexOf(interactionId) !== -1`

Reply:
Done

Reply:
Also added a flag.

Reply:
Thanks for the comment @seanlip, I was expecting this to be resolved as stated [here](https://github.com/oppia/oppia/pull/6926#discussion_r296368758) in the comment.

Reply:
Hm yeah. @anubhavsinha98 please make sure to address all comments, or respond to them clearly ("Okay" is unclear).

Reply:
Sorry @seanlip, will take care of it.